docker-storage-setup.sh

```diff
@@ -35,6 +35,39 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
+usage() {
```

Comment:
We would need to refactor the code into a series of functions.  The problem now is that everything is not a function.

We need these global variables set before they are used.